tests/range/object-expression2.js

```diff
@@ -0,0 +1,9 @@
+
+const y =       [
+    {
+                a: 1,
+    },<<<PRETTIER_RANGE_START>>>
+    {
+              a: 1,
```

Comment:
Could you add an additional property to this object? Otherwise it could be treated as a block + label.

Reply:
Right, it's certainly clearer than trying to find a semicolon `{a: 1;}`. 
Done.